src/odemis/acq/move.py

```diff
@@ -34,10 +34,10 @@
 from odemis.util import executeAsyncTask
 
 MAX_SUBMOVE_DURATION = 60  # s
-LOADING, IMAGING, TILTED, COATING, IMAGING_PATH, COATING_PATH, UNKNOWN = 0, 1, 2, 3, 4, 5, 6
+LOADING, IMAGING, TILTED, COATING, LOADING_PATH, UNKNOWN = 0, 1, 2, 3, 4, 5
```

Comment:
As the coating position is now in the imaging range, returned it back to only one path `LOADING_PATH`